libs/pilight/protocols/433.92/hideki.c

```diff
@@ -0,0 +1,509 @@
+/*
+	Copyright (C) 2018
+
+	This file is part of pilight.
+
+	pilight is free software: you can redistribute it and/or modify it under the
+	terms of the GNU General Public License as published by the Free Software
+	Foundation, either version 3 of the License, or (at your option) any later
+	version.
+
+	pilight is distributed in the hope that it will be useful, but WITHOUT ANY
+	WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+	A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with pilight. If not, see	<http://www.gnu.org/licenses/>
+*/
+
+/*
+ *  this implementation is based on RTL_433 project code for HIDEKI
+ * */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <math.h>
+
+#include "../../core/pilight.h"
+#include "../../core/common.h"
+#include "../../core/dso.h"
+#include "../../core/log.h"
+#include "../protocol.h"
+#include "../../core/binary.h"
+#include "../../core/gc.h"
+#include "hideki.h"
+
+#define MIN_PULSE_LENGTH			220
+#define MAX_PULSE_LENGTH			280
+
+#define MIN_RAW_LENGTH			130
+#define MAX_RAW_LENGTH		180
+#define PULSE_HIDEKI_WEATHER_LOWER		100	// experimentally detected
+#define PULSE_HIDEKI_WEATHER_UPPER		700	// experimentally detected
+
+#define HIDEKI_MAX_BYTES_PER_ROW 14
```

Comment:
Check indenting here.

Reply:
So, I adjusted the formatting on various locations. Do I need to repeat the pull request for these changes?